main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -871,6 +876,15 @@ public void onUpgrade(final SQLiteDatabase db, final int oldVersion, final int n
                             Log.e("Failed to upgrade to ver. 72", e);
                         }
                     }
+                    // Adds coord calculator state to the waypoint
+                    if (oldVersion < 73) {
+                        try {
+                            db.execSQL("ALTER TABLE " + dbTableWaypoints + " ADD COLUMN calc_state TEXT");
+                            db.execSQL("UPDATE " + dbTableWaypoints + " SET calc_state = NULL");
```

Comment:
I think this is not needed as the new column will be initialized to `NULL` by default.